### DISCUSSION TITLE
API Question

### DISCUSSION THREAD

**Discussion by SirEndii** [cc-tweaked/CC-Tweaked#718] [Q&A] [answered]
Hey,
I have a peripheral block which I can connect to a Wired modem, now I have to somehow get information about blocks from the computer network to which my block is connected. E.g. I want to get information about the peripheral "minecraft:chest_0" to store items there. I once looked into the "ComputerCraftAPI" class, but found nothing helpful except maybe "getWiredElementAt". 

I want to push/remove items from a chest with my Me Bridge(See [here](https://docs.srendi.de/Peripherals/mebridge/)). Like `meBridge.importItemFromChest(String item, int count, String chestPeripheral)`.

I hope you can understand what I mean. 🙂 

## Useful information to include:
 - Minecraft version: 1.16.5
 - CC: Tweaked version: 1.94.0

**Comment by SquidDev** [comment, marked as answer]
The best thing to do is probably have a look at what CC:T's inventory methods do:

https://github.com/SquidDev-CC/CC-Tweaked/blob/1825f67eee86b74542adc374b83ed55ba1c5ab68/src/main/java/dan200/computercraft/shared/peripheral/generic/methods/InventoryMethods.java#L133-L143

Basically you want to use `IComputerAccess.getAvailablePeripheral` to get peripherals on the network, and then `IPeripheral.getTarget` to get the original `TileEntity` (well, assuming mods have correctly implemented the method).

**Reply by SirEndii** [reply]
Thank you! It works perfect
